test/ShapeCrawler.Tests.Load/ShapeCollectionTests.cs

```diff
@@ -7,7 +7,6 @@ public class ShapeCollectionTests : SCTest
 {
     [Test]
     [Repeat(40)]
-    [Explicit("Flaky test. Read https://github.com/ShapeCrawler/ShapeCrawler/issues/883")]
     public void AddPicture_should_not_duplicate_the_image_source_When_the_same_svg_image_is_added_to_a_loaded_presentation()
```

Comment:
As @jcoliz mentioned [here](https://github.com/ShapeCrawler/ShapeCrawler/issues/883#issuecomment-2661039496), then do we need in this test and in `AddPicture_should_not_duplicate_the_image_source_When_the_same_svg_image_is_added_twice` also use `Task.Delay(1000).Wait()` instead of the atrribute `Repeat(40)`?

Reply:
I have merged the two normal svg tests and replaced the repeats with Task.Delay